### PR TITLE
Some little fix on RxExamples for XC 8.3

### PR DESCRIPTION
**Pull request by carlosypunto**

### CODE REVIEW

File: RxExample/RxExample/Examples/APIWrappers/APIWrappersViewController.swift [hunk 1]
```diff
@@ -139,7 +139,7 @@ class APIWrappersViewController: ViewController {
 
         // also test two way binding
         let textValue = Variable("")
-        _ = textField.rx.textInput <-> textValue
+        (textField.rx.textInput <-> textValue).disposed(by: disposeBag)
```
Comment (kzaher) [inline RxExample/RxExample/Examples/APIWrappers/APIWrappersViewController.swift]: Hi @carlosypunto ,

We don't need `.disposed(by: disposeBag)` in this case and on line 164. I'm using this code to test will `<->` work without dispose bags.

File: RxExample/RxDataSources/DataSources/Differentiator.swift [hunk 2]
```diff
@@ -73,7 +73,7 @@ struct SectionAssociatedData {
 extension SectionAssociatedData : CustomDebugStringConvertible {
     var debugDescription: String {
         get {
-            return "\(event), \(indexAfterDelete)"
+            return "\(event), \(String(describing: indexAfterDelete))"
```
Comment (kzaher) [inline RxExample/RxDataSources/DataSources/Differentiator.swift]: Could you also please make these changes in RxDataSources project?

Comment (carlosypunto) [inline RxExample/RxDataSources/DataSources/Differentiator.swift]: ok

File: RxExample/RxExample/Examples/APIWrappers/APIWrappersViewController.swift (hunk 1 above)
Comment (carlosypunto) [inline RxExample/RxExample/Examples/APIWrappers/APIWrappersViewController.swift]: I see, then I think those disposables is not being disposed when dismissed. _View controller disposed with 392 resources
assertion failed: Resources weren't cleaned properly, 98 remained, 84 expected_

File: RxExample/RxDataSources/DataSources/Differentiator.swift (hunk 2 above)
Comment (carlosypunto) [inline RxExample/RxDataSources/DataSources/Differentiator.swift]: That was already there

File: RxExample/RxExample/Examples/APIWrappers/APIWrappersViewController.swift (hunk 1 above)
Comment (kzaher) [inline RxExample/RxExample/Examples/APIWrappers/APIWrappersViewController.swift]: Hi @carlosypunto ,

after our talk, I think that previous version should be ok for now. Could you please revert these changes, so I can merge this?

**Pull request merged**